traced/ykrt/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2018"
 license = "Apache-2.0 OR MIT"
 
 [dependencies]
+num_cpus = "1"
```

Comment:
I just checked if this copes with offline CPUs due to cgroups etc. It does:
https://docs.rs/num_cpus/1.13.0/num_cpus/fn.get.html